src/Stryker.CLI/Stryker.CLI/StrykerCLI.cs

```diff
@@ -154,7 +154,7 @@ private void HandleStrykerRunResult(StrykerOptions options, StrykerRunResult res
 
                 Console.WriteLine(Output.Green(" Looks like you've got some work to do :)"));
 
-                ExitCode = 1;
+                ExitCode = 42;
```

Comment:
I don't think a breaking mutation score is the answer the everything, so perhaps we should pick something else :p 

```suggestion
                ExitCode = 2;
```

I would also like to make this less magic. Perhaps a static class with exit codes with clear names describing what they are used for?

Reply:
Actually, yes. Code is the best docs. Hang on!

Reply:
I've updated this. Sorry I didn't mean to be too funny. It's broken window theory - you see one magic number and create another one :facepalm:

Reply:
Haha no problem I can appreciate the humor, just couldn't let it actually get merged like that ;) I was tempted though :)

Reply:
Sure :D